### PR TITLE
templates for new tools env

### PR DESCRIPTION
**Pull request by anilsingla**
templates for new tools env

### CODE REVIEW

File: counterfit/0.0.1/besman-counterfit-BT-env-config.yaml
Comment (asa1997) [inline counterfit/0.0.1/besman-counterfit-BT-env-config.yaml:1]: The config file is for fastjson not for counterfit.

File: counterfit/0.0.1/besman-counterfit-RT-env-config.yaml
Comment (asa1997) [inline counterfit/0.0.1/besman-counterfit-RT-env-config.yaml:1]: The config file is for fastjson not for counterfit.

File: criticalityscore/0.0.1/besman-criticalityscore-BT-env-config.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+BESMAN_ORG: Be-Secure
+
+#  project/ml model/training dataset
+BESMAN_ARTIFACT_TYPE: project 
+
+# Name of the artifact under assessment.
+BESMAN_ARTIFACT_NAME: criticalityscore
+
+# Version of the artifact under assessment.
+BESMAN_ARTIFACT_VERSION: 1.2.24
```
Comment (asa1997) [inline criticalityscore/0.0.1/besman-criticalityscore-BT-env-config.yaml:19]: criticality_score doesnt have a version with tag `1.2.24`

File: criticalityscore/0.0.1/besman-criticalityscore-BT-env-config.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+# The remote repo where we store the assessment reports.
+BESMAN_ASSESSMENT_DATASTORE_URL: https://github.com/Be-Secure/besecure-assessment-datastore
+
+# The path where we download the ansible role of the assessment tools and other utilities
+BESMAN_ANSIBLE_ROLES_PATH: $BESMAN_DIR/tmp/$BESMAN_ARTIFACT_NAME/roles
+
+# The list of tools you wish to install. The tools are installed using ansible roles.# To get the list of ansible roles run 
+#   $ bes list --role
+#add the roles here. format - <Github id>/<repo name>,<Github id>/<repo name>,<Github id>/<repo name>,...
+BESMAN_ANSIBLE_ROLES: Be-Secure/ansible-role-bes-java,Be-Secure/ansible-role-oah-maven,Be-Secure/ansible-role-oah-eclipse
```
Comment (asa1997) [inline criticalityscore/0.0.1/besman-criticalityscore-BT-env-config.yaml:52]: Are these the correct tools for criticality_score?

File: criticalityscore/0.0.1/besman-criticalityscore-RT-env-config.yaml
Comment (asa1997) [inline criticalityscore/0.0.1/besman-criticalityscore-RT-env-config.yaml:1]: The config file is for fastjson not for criticality_score.

File: fossology/0.0.1/besman-fossology-BT-env-config.yaml
Comment (asa1997) [inline fossology/0.0.1/besman-fossology-BT-env-config.yaml:1]: The config file is for fastjson not for fossology.

File: fossology/0.0.1/besman-fossology-RT-env-config.yaml
Comment (asa1997) [inline fossology/0.0.1/besman-fossology-RT-env-config.yaml:1]: The config file is for fastjson not for fossology.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@anilsingla Please update the config file to match the requirements of the tool under test.

File: counterfit/0.0.1/besman-counterfit-BT-env-config.yaml
Comment (anilsingla) [reply counterfit/0.0.1/besman-counterfit-BT-env-config.yaml:1]: done

File: counterfit/0.0.1/besman-counterfit-RT-env-config.yaml
Comment (anilsingla) [reply counterfit/0.0.1/besman-counterfit-RT-env-config.yaml:1]: correctred

File: criticalityscore/0.0.1/besman-criticalityscore-BT-env-config.yaml (hunk 1 above)
Comment (anilsingla) [reply criticalityscore/0.0.1/besman-criticalityscore-BT-env-config.yaml]: corrected

File: criticalityscore/0.0.1/besman-criticalityscore-RT-env-config.yaml
Comment (anilsingla) [reply criticalityscore/0.0.1/besman-criticalityscore-RT-env-config.yaml:1]: corrected

File: fossology/0.0.1/besman-fossology-RT-env-config.yaml
Comment (anilsingla) [reply fossology/0.0.1/besman-fossology-RT-env-config.yaml:1]: done

File: fossology/0.0.1/besman-fossology-BT-env-config.yaml
Comment (anilsingla) [reply fossology/0.0.1/besman-fossology-BT-env-config.yaml:1]: corrected

File: criticalityscore/0.0.1/besman-criticalityscore-BT-env-config.yaml (hunk 2 above)
Comment (anilsingla) [reply criticalityscore/0.0.1/besman-criticalityscore-BT-env-config.yaml]: corrected

**Pull request closed without merging**